src/init/starship.ion

```diff
@@ -4,7 +4,8 @@ fn PROMPT
     let STARSHIP_ION_STATUS = $?
     # Save the ION_CMD_DURATION before running any other commands, but after the status since the
     # duration is not updated during variable assignment
-    let STARSHIP_ION_CMD_DURATION = $ION_CMD_DURATION
+    let STARSHIP_ION_CMD_DURATION = $or($CMD_DURATION $ION_CMD_DURATION)
```

Comment:
I think we should probably add a comment here explaining why we have both `$CMD_DURATION` and  `$ION_CMD_DURATION`.